src/Controller/WebserviceController.php

```diff
@@ -178,7 +178,11 @@ public function webonyxAction(
             $this->eventDispatcher->dispatch($event, ExecutorEvents::PRE_EXECUTE);
 
             if ($event->getRequest() instanceof Request) {
-                $variableValues = $event->getRequest()->request->get('variables', $variableValues);
+                $variableValues = $event->getRequest()->request->get('variables');
+            }
+
+            if (empty($variableValues)) {
```

Comment:
Variable is always set
```suggestion
            if (!$variableValues) {
```